tests/lava/proc/lif/test_models.py

```diff
@@ -15,7 +15,7 @@
 from lava.magma.core.run_configs import RunConfig
 from lava.magma.core.run_conditions import RunSteps
 from lava.magma.core.sync.protocols.loihi_protocol import LoihiProtocol
-from lava.proc.lif.process import LIF
+from lava.proc.lif.process import LIF, TernaryLIF
 
```

Comment:
A couple of comments for the whole file: 
-All tests are too complex to understand why they fail if they do. 
-I can see the logic behind them. But ideally, one would be able to quickly identify the root-cause of a test failing based on the "unit" it tests. Such unit would ideally have a single responsibility and thus debugging becomes "trivial". 
-Perhaps the tests can be refactored (may be in a following issue) to a finer granularity. That would also help in understand whether they are, or not, exhaustive.